x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
+          "after": "",
+          "actions": {
+            "allocate": {
+              "number_of_replicas": "",
+              "include": {},
+              "exclude": {},
+              "require": {
+                "_name": ""
+              }
+            },
+            "shrink": {
+              "number_of_shards": ""
+            },
+            "forcemerge": {
+              "max_num_segments": ""
+            }
+          }
+        },
+        "cold": {
+          "after": "",
+          "actions": {
+            "allocate": {
+              "number_of_replicas": "",
+              "include": {},
+              "exclude": {},
+              "require": {
+                "_name": ""
```

Comment:
why does this has name while included/exclude don't? maybe introduce a global autocomplete for allocation filters that include stuff like `_name` and `_ip` and link to it from all three?

Reply:
added proper params to all.  Since this is the only place allocation comes up, didn't think it worth adding a global.

Reply:
allocation filtering is also in settings FYI